src/store/Errors.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  ReactNode,
+  useEffect,
+  useState,
+  createContext,
+  useContext,
+} from 'react';
+
+type UErrorsContext = {
+  errorMessage: string;
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
avoid exposing such specific types, unless you need it

```suggestion
  setErrorMessage: (value: string) => void;
```

Reply:
Don`t understand this moment.. the setState  function has React.Dispatch<React.SetStateAction<string>> type, but we can use (value: string) => void type for it?